README.md

```diff
@@ -6,7 +6,6 @@
 
 ## Quick start on a fresh project (empty directory)
 ```bash
-yarn init
```

Comment:
this line's not needed... if you `yarn add` and a `package.json` and `yarn.lock` file don't exist, one will be created.